app/middlewares/users.js

```diff
@@ -0,0 +1,39 @@
+const { validationResult, check } = require('express-validator'),
+  error = require('../errors'),
+  logger = require('../logger');
+
+const validateRequest = (req, res, next) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    logger.error(errors.errors);
```

Comment:
in this case in the console print `[object object]`, if you want print the message you hove to errors.errors[0]